grafana-cloud-metrics.yaml

```diff
@@ -304,6 +304,10 @@ resources:
             url=https://monitoring.api.rackspacecloud.com/v1.0
             EOL
             start graphite-api
+            ufw enable
+            ufw allow ssh
+            ufw allow http
+            ufw allow https
           params:
```

Comment:
Let me know if there's a cleaner/more concise way to open up the 3 port instead of having 3 separate allow calls.  I did some research and did not see anyone doing anything other than allowing for an ip range.